plugins/dns/plugin.go

```diff
@@ -18,12 +18,22 @@ func init() {
 }
 
 var (
-	dnsServers []net.IP
+	dnsServers6 []net.IP
+	dnsServers4 []net.IP
 )
 
 func setupDNS6(args ...string) (handler.Handler6, error) {
-	// TODO setup function for IPv6
-	log.Warning("not implemented for IPv6")
+	if len(args) < 1 {
```

Comment:
How about having a more generic `setupDNS(v6 bool, args ...string)` to avoid code dup. Similar to:
https://github.com/coredhcp/coredhcp/blob/master/plugins/file/plugin.go#L197

Reply:
Looks like this:
```go
func setupDNS(v int, args ...string) (handler.Handler4, handler.Handler6, error) {
	log.Printf("loaded plugin for DHCPv%d.", v)
	if len(args) < 1 {
		return nil, nil, errors.New("need at least one DNS server")
	}
	for _, arg := range args {
		DNSServer := net.ParseIP(arg)
		if v == 4 {
			if DNSServer.To4() == nil {
				return Handler4, nil, errors.New("expected an DNS server address, got: " + arg)
			}
			dnsServers4 = append(dnsServers4, DNSServer)
			log.Infof("loaded %d DNS servers.", len(dnsServers4))
		} else {
			if DNSServer.To16() == nil {
				return nil, Handler6, errors.New("expected an DNS server address, got: " + arg)
			}
			dnsServers6 = append(dnsServers6, DNSServer)
			log.Infof("loaded %d DNS servers.", len(dnsServers6))
		}
	}
	return Handler4, Handler6, nil
}
```

To me it doesn't really look better (code duplication is just moved into ifs instead of at the function level mostly), but I'm not attached to one or the other; if you think this is better I'll put it in.

Reply:
> To me it doesn't really look better

Ok, let's leave it like this for now then.